Makefile

```diff
@@ -10,11 +10,13 @@ build: test
 
 lint:
 	@echo "lint"
-	@gometalinter ./...
+	@golangci-lint run ./...
+	@find . -type f -not -path "./vendor/*" -print0 | xargs -0 misspell
 
 lint-install:
-	go get -u github.com/alecthomas/gometalinter
-	gometalinter --install
+	go get -u github.com/client9/misspell/cmd/misspell
```

Comment:
hi!
starting from v1.8 golangci-lint supports misspell and you no longer need to install it :)

Reply:
Hi @jirfag that's great to hear, thanks for the tip!